src/app.py

```diff
@@ -820,20 +821,26 @@ def app_upgrade(app=[], url=None, file=None, force=False, no_safety_backup=False
             # If upgrade failed or broke the system,
             # raise an error and interrupt all other pending upgrades
             if upgrade_failed or broke_the_system:
```

Comment:
I'm wondering if we shouldnt still *not* continue if `broke_the_system is True` ... like, this means that either dpkg is broken, or nginx is broken, which has a 95% change of breaking the next upgrade (whereas the `upgrade_failed is True and broke_the_system is False` scenario is more likely to be specific to the app)

Reply:
Ah yeah make totally sens :)